### DISCUSSION TITLE
How do you pass environment variables to a component test?

### DISCUSSION THREAD

**Discussion by gvocale** [cypress-io/cypress#21031] [Component Testing]
I have tried the following, but doesn't seem to work:

```
yarn cypress open-ct --env IS_TEST=true
```

and then in a react component
```tsx
console.log(process.env.IS_TEST) //undefined
```

How do I pass the `IS_TEST` variable so that I can access it in a React component?

**Comment by geromegrignon** [comment]
Hi @gvocale,

the `--env` option must be used on `cypress run`: https://docs.cypress.io/guides/guides/environment-variables#Option-4-env

**Comment by alessandro308** [comment]
I tried to 
- add --env
- add defineConfig({component.env})
- add .env.test
but still get undefined in the component if I console.log the value. Any idea?
